windows/runner/win32_window.h

```diff
@@ -0,0 +1,98 @@
+#ifndef RUNNER_WIN32_WINDOW_H_
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase